operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -237,12 +237,14 @@ spec:
 # service: The Kubernetes service name for tracing. Kiali uses this to connect within the cluster to Jaeger.
 # url: The URL that Kiali uses when integrating with Tracing. This URL must be accessible to clients external to
 #      the cluster in order for the integration to work properly. If empty, an attempt to auto-discover it is made.
+# query_base_path: The value stored in QUERY_BASE_PATH in tracing service
 #    ---
 #    tracing:
 #      enabled: true
 #      namespace: ""
 #      service : ""
 #      url: ""
+#      query_base_path: ""
```

Comment:
all of these settings are in alphabetical sort order. Move `query_base_path` after `namespace` and before `service`
Do the same with the comment line above.

Ex:

```
# query_base_path: The value stored in QUERY_BASE_PATH in tracing service
# service: The Kubernetes service name for tracing. Kiali uses this to connect within the cluster to Jaeger.
# url: The URL that Kiali uses when integrating with Tracing. This URL must be accessible to clients external to
#      the cluster in order for the integration to work properly. If empty, an attempt to auto-discover it is made.
#    ---
#    tracing:
#      enabled: true
#      namespace: ""
#      query_base_path: ""
#      service : ""
#      url: ""
```